getting-started/templates/systemlink-admin-values.yaml

```diff
@@ -26,12 +26,14 @@ strimzi-kafka-operator:
   watchAnyNamespace: true
   podSecurityContext: 
     runAsNonRoot: true
-  ## Uncomment to set Java heap size
-  ## Only required if the operator is crashing with
-  ## "Terminating due to java.lang.OutOfMemoryError: Java heap space"
-  #extraEnvs:
-  #  - name: JAVA_OPTS
-  #    value: "-Xms256m -Xmx256m"
+  extraEnvs:
```

Comment:
Doing this in the template only helps if the use updates their configuration to match the template. If we want this to be the default, we should change the chart (either the leaf chart of the top-level chart values file). The template should be fore configuration that the user needs to customize.

Reply:
Ah that's right. I already have a PR for that but the pipeline is paused. I'll add owners over there once the pipeline is unpaused. I'll abandon this PR.

Reply:
Does this need to be changed for the current release? If so, talk to Christian.